Sources/Prometheus/PrometheusMetricsFactory.swift

```diff
@@ -16,8 +16,16 @@ import CoreMetrics
 
 /// A wrapper around ``PrometheusCollectorRegistry`` to implement the `swift-metrics` `MetricsFactory` protocol
 public struct PrometheusMetricsFactory: Sendable {
-    /// The underlying ``PrometheusCollectorRegistry`` that is used to generate
-    public var client: PrometheusCollectorRegistry
+    private static let _defaultRegistry = PrometheusCollectorRegistry()
+
+    /// The default ``PrometheusCollectorRegistry``, which is used inside the ``PrometheusMetricsFactory``
+    /// if no other is provided in ``init(client:)`` or set via ``PrometheusMetricsFactory/client``
+    public static var defaultRegistry: PrometheusCollectorRegistry {
+        self._defaultRegistry
+    }
+
+    /// The underlying ``PrometheusCollectorRegistry`` that is used to generate the swift-metrics handlers
+    public var registry: PrometheusCollectorRegistry
```

Comment:
Do we want this to be public?

Reply:
Yes. It should! So that users can access what they got by default.

Reply:
I don't mind I guess :)